File: 240309/가장 큰 수 고르기/picking-biggest-number.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    int max_val=0;
    int arr[10]={};

    for(int i=0;i<10;i++){
        cin>>arr[i];
        if(arr[i]>max_val) max_val=arr[i];
    }
    cout<<max_val;
    return 0;
}

File: 240606/가장 큰 수 고르기/picking-biggest-number.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
    Scanner sc=new Scanner(System.in);

    int[] arr=new int[10];

    for(int i=0;i<10;i++){
        arr[i]=sc.nextInt();
    }

    int maxval=arr[0];

    for(int i=1;i<10;i++){
        if(arr[i]>maxval)
        maxval=arr[i];
    }
    System.out.print(maxval);
    
    }
}